.travis.yml

```diff
@@ -28,20 +28,30 @@ matrix:
         - SO_S3_URL: "s3://smart-open-py36-benchmark"
         - SO_S3_RESULT_URL: "s3://smart-open-py36-benchmark-results"
 
+    - python: '3.7'
+      env:
+        - SO_DISABLE_MOCKS: "1"
+        - SO_S3_URL: "s3://smart-open-py37-benchmark"
+        - SO_S3_RESULT_URL: "s3://smart-open-py37-benchmark-results"
+      dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
+      sudo: required  # required for Python 3.7 (travis-ci/travis-ci#9069)
+
 install:
   - pip install .[test]
   - pip uninstall --yes botocore boto3  # workaround for avoid very similar conflict as in https://github.com/venth/aws-adfs/issues/52
-  - pip install boto3
+  - pip install boto3 flake8
   - pip freeze
 
-
 script:
   - if [[ ${TRAVIS_SECURE_ENV_VARS} = false ]]; then
       echo "DISABLE INTEGRATION TESTING FOR S3";
       unset SO_DISABLE_MOCKS;
       unset SO_S3_URL;
       unset SO_S3_RESULT_URL;
     fi
+  - if  [[ $TRAVIS_PYTHON_VERSION = '*.7' ]]; then
```

Comment:
@cclauss no need to add flake here, this will be added with other changes & code cleanup later